backend/src/routes/books.ts

```
@@ -0,0 +1,116 @@
+ * @route PUT /:id
+ * @desc ID로 책 수정
+ * @access Admin
+ */
+router.put(
+  '/:id',
+  async (
+    req: Request<{ id: string }, {}, { title?: string; author?: string }>,
+    res: Response
+  ) => {
+    try {
+      const book = await Book.findByIdAndUpdate(req.params.id, req.body, {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)